src/custom/pages/Claim/CanUserClaimMessage.tsx

```diff
@@ -14,7 +14,16 @@ export default function CanUserClaimMessage({ hasClaims, isAirdropOnly }: ClaimI
   const { activeClaimAccount, claimStatus } = useClaimState()
   const { setActiveClaimAccount } = useClaimDispatchers()
 
-  const end = useAirdropDeadline()
+  const { airdropDeadline, deployment, investmentDeadline, isAirdropWindowOpen, isInvestmentWindowOpen } =
```

Comment:
Yeah, I've based all of those PRs on the same branch so this update did not have that.
Anyway, here we dont need all of those vars AFAICT, so loading only `airdropDeadline` is enough

I can take care of it